ccmlib/node.py

```diff
@@ -1006,11 +1006,11 @@ def clear(self, clear_all=False, only_data=False, saved_caches=False):
             if only_data:
                 for dir in os.listdir(full_dir):
                     keyspace_dir = os.path.join(full_dir, dir)
-                    if os.path.isdir(keyspace_dir) and dir != "system":
-                        for f in os.listdir(keyspace_dir):
-                            full_path = os.path.join(keyspace_dir, f)
-                            if os.path.isfile(full_path):
-                                os.remove(full_path)
+                    if os.path.isdir(keyspace_dir) and not dir.startswith("system"):
```

Comment:
does this change apply to https://github.com/pcmanus/ccm as well, probably we can upstream this change?

Reply:
I'm not exactly sure where the  upstream is

Since they are using personal forks

I don't think there is a real one for that

Reply:
I think we forked from this project. And I am sure this is the upstream.

Reply:
It's moved

And that the name I've seen mentioned in the upstream dtest repo
https://github.com/riptano/ccm

We could suggest it, but I don't know if they would care about it or not.

I still don't think it's needed, we aren't gonna sync from it, maybe only cherry pick specific things.

Reply:
No it’s not needed. The reason why suggest this was not because it’s needed, but because it’s a community, we benefit from others‘ work, would be ideal if we can contribute back.

Reply:
I would suggest opening an issue. and see if anyone is interested in this getting fixed.

My past experiences in other similar cases, it was a bit tedious, and took a long time to get things in.

I don't say we shouldn't do those, but just be mindful of people time (and not block thier PRs for those)